core/server/master/src/main/java/alluxio/master/metastore/caching/Cache.java

```diff
@@ -114,35 +113,32 @@ public Cache(CacheConfiguration conf, String name, MetricKey evictionsKey, Metri
    * @return the value, or empty if the key doesn't exist in the cache or in the backing store
    */
   public Optional<V> get(K key, ReadOption option) {
-    try (Timer.Context ctx = MetricsSystem
-        .timer(MetricKey.MASTER_INODE_CACHE_LOAD_TIMER.getName()).time()) {
```

Comment:
Is this a revert?
Can you remove the metric key if so?

Reply:
done